src/universal/modules/teamDashboard/components/TeamSettings/TeamSettings.js

```diff
@@ -148,7 +153,17 @@ const TeamSettings = (props) => {
                 actions={orgApprovalRowActions(orgApproval)}
               />
             );
-          })
+          })}
+          {myTeamMember.isLead &&
+            <Panel label="Danger Zone">
+              <div className={css(styles.dangerZoneChildren)}>
+                <ArchiveTeamConfirmation
+                  teamId={team.id}
+                  teamName={team.name}
+                  router={router}
```

Comment:
instead of passing in a router, let's just use the `withRouter` HOC on the `ArchiveTeamConfirmation` component. That way, we decouple the 2 components a bit more (ie we could remove the router from this component, or use the child component somewhere where the parent doesn't have the router).